app/routes/docs_.schema.$version.$/components.tsx

```diff
@@ -145,7 +145,8 @@ export function formatFieldName(name: string, requiredProps?: string[]) {
 }
 
 export function formatFieldType(item: SchemaProperty): string {
-  if (item.type) return item.type
+  if (item.type)
+    return typeof item.type == 'object' ? item.type.join(' | ') : item.type
```

Comment:
Fewer nested `if` statements:

```suggestion
  if (item.type === 'object') return item.type.join(' | ')
  if (item.type) return item.type
```